config/eslintrc_react.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
         'react/no-is-mounted': 2,
         'react/no-multi-comp': 0, // Enable to define a multiple component to a single file.
         'react/no-render-return-value': 2,
+        'react/no-redundant-should-component-update': 0, // we'll reconsider this option when we begin to use `PureComponent`.
```

Comment:
[`PureComponent`](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) do shallow compare for props and state, but a stateless component does not have such optimization (at least I remember so it). A stateless component is a component having only `props` and it does not have any backstore instance for component to keep component's state.